sdk/trace/src/test/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessorTest.java

```diff
@@ -45,7 +46,14 @@
   private static final long MAX_SCHEDULE_DELAY_MILLIS = 500;
   private SdkTracerProvider sdkTracerProvider;
   private final BlockingSpanExporter blockingSpanExporter = new BlockingSpanExporter();
-  @Mock private SpanExporter mockServiceHandler;
+
+  @Mock(lenient = true)
+  private SpanExporter mockServiceHandler;
+
+  @BeforeEach
+  void setUp() {
+    when(mockServiceHandler.shutdown()).thenReturn(CompletableResultCode.ofSuccess());
```

Comment:
Maybe add to the test method instead of using lenient?

Reply:
I can look into it. It wasn't actually clear what test needed this. :)

Reply:
I figured it out. This is one reason I don't like having mocks as fields if they're not used in every test. :(

Reply:
From what I understand it's the use of `when` in beforeeach that triggers the strict checking, not the presence of a field. But no worries either way.